src/components/Session/Chat.vue

```diff
@@ -1,79 +1,112 @@
 <template>
-  <div class="chat">
-    <div class="header">Chat</div>
-    <div class="messages-container">
-			<div class="messages">
-				<template v-for="message in messages">
-					<div class="message" v-bind:class="leftRightMessage(message)">
-						<div class="avatar" v-bind:style="message.avatarStyle"></div>
-						<div class="contents">
-              <div class="name">
-								{{message.name}}
-							</div>
-							{{message.contents}}
-              <div class="time">
-								{{message.time}}
-							</div>
+<div class="chat">
+  
+  <div class="header">Chat</div>
+
+  <div class="message-box">
+
+    <div class="chat-warning">
+      Messages cannot contain personal information
+      <span class="chat-warning__close" @click="hideModerationWarning">×</span>
+    </div>
+
+		<div class="messages">
+			<template v-for="message in messages">
+
+				<div class="message" :class="message.name === user.firstname ? 'left' : 'right'">
+					<div class="avatar" :style="message.avatarStyle"></div>
+					<div class="contents">
+            <div class="name">
+							{{message.name}}
+						</div>
+						{{message.contents}}
+            <div class="time">
+							{{message.time}}
 						</div>
 					</div>
-				</template>
-			</div>
-    </div>
-    <textarea v-on:keyup.enter="sendMessage" v-model="newMessage" placeholder="Type here."></textarea>
+				</div>
+
+			</template>
+		</div>
   </div>
+
+  <textarea @keyup.enter="sendMessage" v-model="newMessage" placeholder="Type here..."></textarea>
+
+</div>
 </template>
 
-<script>
 
-import $ from 'jquery'
-import moment from 'moment'
+<script>
+import moment from 'moment';
 
-import UserService from 'src/services/UserService'
-import SessionService from 'src/services/SessionService'
+import UserService from 'src/services/UserService';
+import SessionService from 'src/services/SessionService';
+import ModerationService from 'src/services/ModerationService';
 
 const DEFAULT_AVATAR_URL = 'static/defaultAvatar@2x.png';
 
 export default {
-	data(){
-    var user = UserService.getUser();
+	data() {
 		return {
-      user: user,
+      user: UserService.getUser(),
       messages: [],
 			currentSession: SessionService.currentSession,
 			newMessage: ''
 		}
 	},
+
   methods: {
-    sendMessage: function() {
-      var message = this.newMessage;
+    showModerationWarning() {
+      document.querySelector('.chat-warning').style.top = 0;
```

Comment:
You should use `v-show` and set a variable when the chat warning should be shown. Do not use CSS for that.